### ISSUE TITLE
Is there a way to specify number of cores for evosuite test generation using commandline 

### ISSUE THREAD

**Issue by firetorch** [EvoSuite/evosuite#25]

**Comment by arcuri82** [comment]
Not really... on a single class under test (CUT), currently there is no way to use more than one core, and adding such feature is not planned for any foreseeable future :(

however, the Maven and IntelliJ plugins use a special mode that allows to test several CUTs in parallel.  This is possible to do from command line, but it requires quite a few parameters and several things to consider (eg side-effects). And more importantly, it is not properly documented. You can have a look at:

https://github.com/EvoSuite/evosuite/blob/2bc8e1b44c5f30767802a63c41f124cc72691802/plugins/maven/src/main/java/org/evosuite/maven/GenerateMojo.java

to see how the Maven plugin starts EvoSuite for multi-core. 

We could add the documentation to explain how to do it from command line, but, what would be the business case? I mean, how are you using EvoSuite? (eg, I assume you are using neither Maven or IntelliJ)

cheers

Andrea

**Comment by firetorch** [comment]
Adding more cores through command line could be useful to run tests on the entire jar containing 2000+ classes.. Currently I see this option available in IntelliJ plugin but not in command line .But using intellij plugin in my case has an Issue I will discuss in separate thread.    I can now generate tests on my jar with mem=6000  for approx 10 hrs and it generates tests for 850 classes with timeout error. Can I do better?
